portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="ravenclaw_stripes">
+    <div id="content">
+        <h1 id="title"><strong>Alisa's Gallery</strong></h1>
+
```

Comment:
Nitpick: Remove the unnecessary blank lines.

Reply:
I committed a change which removes unnecessary blank lines.